02-wrangling.Rmd

```diff
@@ -23,14 +23,13 @@ By the end of the chapter, students will be able to:
     - `group_by`
     - `map`
 
-## Vectors and Data frames
+## Data frames and Vectors
 
-At this point, we know how to load data into R from various file formats. Once loaded into R, most of the tools we have learned about for reading data into R represent the data as a data frame. So now we will spend some time learning more about data frames in R so that we have a better understanding 
-of how we can use and manipulate these objects.
+At this point, we know how to load data into R from various file formats. Once loaded into R, most of the tools we have learned about for reading data into R represent the data as a data frame. So now we will spend some time learning more about data frames in R so that we have a better understanding of how we can use and manipulate these objects.
```

Comment:
```suggestion
We now know how to load data into R from various file formats.
Given that most of the tools we have use to do this represent the data as a 
data frame in R, we will spend some time learning more about data frames
to deepen our understanding of them. This will be useful prerequisite knowledge
to use and manipulate these objects in our data analysis.
```